modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ViewObjectEntriesDisplayContext.java

```diff
@@ -199,7 +199,14 @@ public FDSSortItemList getFDSSortItemList() {
 				FDSSortItemBuilder.setDirection(
 					objectViewSortColumn.getSortOrder()
 				).setKey(
-					objectViewSortColumn.getObjectFieldName()
+					() -> {
+						String fieldName = StringUtil.replace(
```

Comment:
About the parameter name. In this kind of situations, we usually link the class name and method name to build a most meaningful variable name

i. e: ObjectField.getName() = objectFieldName

 But in this situation, the method used already have all information of this variable, so you could admit:
 
 objectViewSortColumn.get**ObjectFieldName**() = **objectFieldName**